src/main/native/compression/IntelInflater.cc

```diff
@@ -78,8 +78,11 @@ JNIEXPORT void JNICALL Java_com_intel_gkl_compression_IntelInflater_resetNative
 
      lz_stream = (inflate_state*)calloc(1,sizeof(inflate_state));
       if ( lz_stream == NULL ) {
-        jclass Exception = env->FindClass("java/lang/Exception");
-        env->ThrowNew(Exception,"Memory allocation error");
+
+        if(env->ExceptionCheck())
+            env->ExceptionClear();
+
```

Comment:
same as above

Reply:
Same as above, we might expect exceptions from malloc and calloc operations like L#79 so this exception check is to handle that.